src/i18n/en/nav.ts

```diff
@@ -13,71 +13,81 @@ export default [
+	{ text: 'Build a Blog', slug: 'tutorial/0-introduction', key: 'blog-tutorial' },
+	// { text: 'Thinking with Islands', slug: 'tutorial/0-introduction', key: 'island-tutorial' },
+
 	{ text: 'Basics', header: true, type: 'learn', key: 'basics' },
+	
 	{
 		text: 'Project Structure',
 		slug: 'core-concepts/project-structure',
 		key: 'core-concepts/project-structure',
 	},
 	{
-		text: 'Astro Components',
+		text: 'Components',
 		slug: 'core-concepts/astro-components',
 		key: 'core-concepts/astro-components',
 	},
 	{ text: 'Pages', slug: 'core-concepts/astro-pages', key: 'core-concepts/astro-pages' },
 	{ text: 'Layouts', slug: 'core-concepts/layouts', key: 'core-concepts/layouts' },
-	{ text: 'Markdown & MDX', slug: 'guides/markdown-content', key: 'guides/markdown-content' },
-	{ text: 'Routing', slug: 'core-concepts/routing', key: 'core-concepts/routing' },
-	{ text: 'Imports', slug: 'guides/imports', key: 'guides/imports' },
-	{ text: 'Endpoints', slug: 'core-concepts/endpoints', key: 'core-concepts/endpoints' },
-	{ text: 'Data Fetching', slug: 'guides/data-fetching', key: 'guides/data-fetching' },
-	{ text: 'Deploy', slug: 'guides/deploy', key: 'guides/deploy' },
-	{ text: 'Troubleshooting', slug: 'guides/troubleshooting', key: 'guides/troubleshooting' },
 
+	{ text: 'Recipes', header: true, type: 'learn', key: 'examples' },
+	{ text: 'Migrate to Astro', slug: 'guides/migrate-to-astro', key: 'guides/migrate-to-astro' },
+	{ text: 'Connect a CMS', slug: 'guides/cms', key: 'guides/cms' },
+	{ text: 'Add integrations', slug: 'guides/integrations-guide', key: 'guides/integrations-guide' },
+	{ text: 'Deploy your site', slug: 'guides/deploy', key: 'guides/deploy' },
+	{ text: 'More Recipes', slug: 'recipes', key: 'guides/recipes' },
+
+
+	// TODO: Can some of these be removed, merged, or pulled entirely into Recipes? (ex: RSS as recipe)
 	{ text: 'Guides', header: true, type: 'learn', key: 'features' },
-	{ text: 'Integrations', slug: 'guides/integrations-guide', key: 'guides/integrations-guide' },
+	{
+		text: 'Astro Template Syntax',
+		slug: 'core-concepts/astro-syntax',
+		key: 'core-concepts/astro-syntax',
+	},
 	{
 		text: 'UI Frameworks',
 		slug: 'core-concepts/framework-components',
 		key: 'core-concepts/framework-components',
 	},
 	{
-		text: 'Scripts & Event Handling',
-		slug: 'guides/client-side-scripts',
-		key: 'guides/client-side-scripts',
+		text: 'Content Collections',
+		slug: 'guides/content-collections',
+		key: 'guides/content-collections',
 	},
-	{ text: 'CSS & Styling', slug: 'guides/styling', key: 'guides/styling' },
 	{
 		text: 'Server-side Rendering (SSR)',
 		slug: 'guides/server-side-rendering',
 		key: 'guides/server-side-rendering',
 	},
-	{ text: 'Authoring Content', slug: 'guides/content', key: 'guides/content' },
 	{
-		text: 'Content Collections',
-		slug: 'guides/content-collections',
-		key: 'guides/content-collections',
+		text: 'Scripts & Event Handling',
```

Comment:
Personal preference would be to keep this close to the CSS & Styling sidebar entry as CSS + JS are two related types of content.